coordinator/src/bin/coordinator.rs

```diff
@@ -128,6 +133,49 @@ async fn main() -> Result<()> {
         }
     });
 
+    tokio::spawn({
+        let node = node.clone();
+        async move {
+            loop {
+                let node = node.clone();
+                spawn_blocking(move || {
+                    let cx = opentelemetry::Context::current();
+                    let channels = node.inner.channel_manager.list_channels();
+                    for channel_detail in channels {
+                        let key_values = [
+                            KeyValue::new("channel_id", hex::encode(channel_detail.channel_id)),
+                            KeyValue::new("is_outbound", channel_detail.is_outbound),
+                            KeyValue::new("is_public", channel_detail.is_public),
+                        ];
+                        CHANNEL_BALANCE_MSATOSHI.observe(
```

Comment:
nit: do we really want to go with `msat` on our metrics? it makes sense for feature-proofing app in certain places, but for our reporting I can't imagine us caring about these fractions (not that they're used anyway).